src/Bicep.Cli/Commands/RootCommand.cs

```diff
@@ -101,6 +101,26 @@ private void PrintHelp()
       bicep decompile file.json --outdir dir1
       bicep decompile file.json --force
       bicep decompile file.json --outfile file.bicep
+
+  {exeName} generate-params [options] <file>
+    Builds .parameters.json file from the given bicep file.
+
+    Arguments:
+      <file>        The input file
+
+    Options:
+      --no-restore      Generates the parameters file without restoring external modules.
+      --outdir <dir>    Saves the output at the specified directory.
+      --outfile <file>  Saves the output as the specified file path.
```

Comment:
For outfile, should this be "Saves the output as the specified file name" -? Would that be more accurate @polatengin

Reply:
I think that makes more sense. but, I was following the same wording with the `build` https://github.com/Azure/bicep/blob/53254fb484b24b5a3746ed01e3cbc5acfcf3e256/src/Bicep.Cli/Commands/RootCommand.cs#L75 and `decompile` https://github.com/Azure/bicep/blob/53254fb484b24b5a3746ed01e3cbc5acfcf3e256/src/Bicep.Cli/Commands/RootCommand.cs#L94 help texts. they have `Saves the output as the specified file path.` for the `--outfile` option.

what you think? should I leave it as-is, only update it for `generate-params` command, or update every instance?

Reply:
Since they can specify a full path (right?), "file path" is more technically correct.  If so, I vote for leave as is.

My assumption is that you can't specify both --outdir and --outfile, correct?

Reply:
yes, they don't work together, one or the other

Reply:
Okay Leave as is